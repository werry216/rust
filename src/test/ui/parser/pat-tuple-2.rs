fn main() {
    match 0 {
        (pat, ..,) => {}
        //~^ ERROR trailing comma is not permitted after `..`
        //~| ERROR mismatched types
    }
}
