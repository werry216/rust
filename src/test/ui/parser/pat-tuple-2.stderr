error: trailing comma is not permitted after `..`
  --> $DIR/pat-tuple-2.rs:3:17
   |
LL |         (pat, ..,) => {}
   |                 ^ trailing comma is not permitted after `..`

error[E0308]: mismatched types
  --> $DIR/pat-tuple-2.rs:3:9
   |
LL |         (pat, ..,) => {}
   |         ^^^^^^^^^^ expected integer, found tuple
   |
   = note: expected type `{integer}`
              found type `(_,)`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0308`.
